### PR TITLE
Set appropriate HTTPS envvar

### LINKED ISSUE #37: Nginx doesn't detect the upstream https connection
AppEngines uses HTTPS almost exclusively so it's a good idea to pin HTTPS
to allow applications proper connection detection based on the current protocol.

### PR DESCRIPTION
**Pull request by tmatsuo**
fixes #37

### CODE REVIEW

File: testapps/php56_custom_configs/nginx-custom.conf [hunk 1]
```diff
@@ -76,11 +81,28 @@ http {
...
 
-	location /hello {
-          return 200 'hello from nginx-custom.conf';
+        # Drop accesses for some files
+        location = /favicon.ico { access_log off; log_not_found off; }
+        location ~ /\.          { access_log off; log_not_found off; deny all; }
+        location ~ ~$           { access_log off; log_not_found off; deny all; }
+
+        location /hello {
+            return 200 'hello from nginx-custom.conf';
```
Comment (bshaffer) [inline testapps/php56_custom_configs/nginx-custom.conf]: Seems like we should remove this?

File: testapps/php56_nginx_conf/nginx.conf [hunk 2]
```diff
@@ -76,11 +81,28 @@ http {
...
         include /etc/nginx/conf.d/*.conf;
 
-	location /hello {
-          return 200 'hello from nginx.conf';
+        # Drop accesses for some files
+        location = /favicon.ico { access_log off; log_not_found off; }
+        location ~ /\.          { access_log off; log_not_found off; deny all; }
+        location ~ ~$           { access_log off; log_not_found off; deny all; }
+
+        location /hello {
```
Comment (bshaffer) [inline testapps/php56_nginx_conf/nginx.conf]: This too. These are examples that shouldn't be shipped!

File: testapps/php56_custom_configs/nginx-custom.conf (hunk 1 above)
Comment (tmatsuo) [inline testapps/php56_custom_configs/nginx-custom.conf]: This is a config file for the test app. The test will access this path to make sure it is working.

File: testapps/php56_nginx_conf/nginx.conf (hunk 2 above)
Comment (tmatsuo) [inline testapps/php56_nginx_conf/nginx.conf]: This is a config for the test app.

**Comment by tmatsuo** [comment]
FYI it's in prod

**Pull request merged**